docs/src/index.md

```diff
@@ -129,9 +129,9 @@ julia> gradient(colordiff, RGB(1, 0, 0), RGB(0, 1, 0))
 ((r = 0.4590887719632896, g = -9.598786801605689, b = 14.181383399012862), (r = -1.7697549557037275, g = 28.88472330558805, b = -0.044793892637761346))
 ```
 
-## Gradients of ML models
+## Explicit and implicit parameters of ML models
 
-It's easy to work with even very large and complex models, and there are few ways to do this. Autograd-style models pass around a collection of weights.
+It's easy to work with even very large and complex models, and there are few ways to do this. Autograd-style models pass around a collection of weights. There are two ways of passing *explicit* parameters:
```

Comment:
I think it should be "parameters for ML models" not "of".

But more importantly, what two ways are you seeing? In my thinking, `gradient(f, x)` which evaluates `f(x)` is one way, whether `x` is a number, a vector, a tuple, a dictionary, a struct -- the point is that it's an argument of the function.

Reply:
I also confess that I don't really know what "Autograd-style" means, exactly. Does this mean Tracker.jl? Some python package everyone was familiar with back in the day?

Reply:
I assume PyTorch's AD called autograd.